assets/components/headers/countrySwitcherHeader/countrySwitcherHeaderContainer.js

```diff
@@ -6,55 +6,28 @@ import { connect } from 'react-redux';
 
 import CountrySwitcherHeader from 'components/headers/countrySwitcherHeader/countrySwitcherHeader';
 
-import type { CountryGroupId } from 'helpers/internationalisation/countryGroup';
+import { countryGroups, type CountryGroupId } from 'helpers/internationalisation/countryGroup';
 import type { CommonState } from 'helpers/page/page';
+import type { PropTypes } from './countrySwitcherHeader';
 
 
-const availableCountriesGroups: CountryGroupId[] =
-  ['GBPCountries', 'UnitedStates', 'EURCountries', 'NZDCountries', 'Canada', 'International', 'AUDCountries'];
-
-// ----- Functions ----- //
-
-function handleCountryGroupChange(value: string): void {
-  switch (value) {
-    case 'UnitedStates':
-      window.location.pathname = '/us/contribute';
-      break;
-    case 'GBPCountries':
-      window.location.pathname = '/uk/contribute';
-      break;
-    case 'EURCountries':
-      window.location.pathname = '/eu/contribute';
-      break;
-    case 'NZDCountries':
-      window.location.pathname = '/nz/contribute';
-      break;
-    case 'Canada':
-      window.location.pathname = '/ca/contribute';
-      break;
-    case 'International':
-      window.location.pathname = '/int/contribute';
-      break;
-    case 'AUDCountries':
-      window.location.pathname = '/au/contribute';
-      break;
-    default:
-  }
-}
+// ------ Component ----- //
 
+export default function (subPath: string, listOfCountries: CountryGroupId[]) {
 
-// ----- State Maps ----- //
+  function handleChange(cg: CountryGroupId): void {
+    window.location.pathname = `/${countryGroups[cg].supportInternationalisationId}${subPath}`;
```

Comment:
maybe a more readable variable name, `countryGroupId` or `cgId`?

Reply:
`cgId` it is.